models/FPG/builder.py

```diff
@@ -0,0 +1,449 @@
+import mxnet as mx
+import mxnext as X
+from symbol.builder import Neck
+from models.NASFPN.builder import NASFPNNeck
+
+def merge_sum(sum_list, name):
```

Comment:
I've seen this two helper functions so many times across the codebase. How about move them in mxnext?